red.js

```diff
@@ -23,6 +23,7 @@ var nopt = require("nopt");
 var path = require("path");
 var fs = require("fs");
 var RED = require("./red/red.js");
+var log = require("./red/log");
```

Comment:
You don't need to require `log` directly. You can already access it as `RED.log`